Dockerfile

```diff
@@ -96,20 +96,23 @@ COPY --from=builder /tmp/wp-src/waypoint-entrypoint /usr/bin/waypoint-entrypoint
 
 VOLUME ["/data"]
 
+# NOTE: userid must be 100 here. Otherwise upgrades prior to 0.2.1 will fail
```

Comment:
Wouldn't changing this at any point break all upgrades from that point backward? like if we're in v0.6.2 and change this, then upgrades from 0.6.1 and prior would fail?

Reply:
Yep, that's true. I'll simplify it to just say don't change it :D haha